src/fparser/Fortran2003.py

```diff
@@ -561,32 +561,51 @@ class BracketBase(Base):
 ::
     <bracket-base> = <left-bracket-base> <something> <right-bracket>
     """
+    @staticmethod
```

Comment:
It was a static method before, I've just changed the way it's declared (using a decorator) instead of the previous:

    match = staticmethod(match)